sdb/commands/zfs/btree.py

```diff
@@ -52,6 +52,8 @@ class Btree(sdb.Walker):
     names = ["zfs_btree"]
     input_type = "zfs_btree_t *"
 
+    elem_size: drgn.Object
```

Comment:
Is this describing a class level variable, or an instance variable? Presumably this is for the instance variable, but it strikes me as odd, since this is where class variables would be defined (e.g. the two above).

Reply:
good catch - should be fixed now.
Take another look at the change and let me know if you find anything else otherwise let's merge this.